DuckDuckGo/Info.plist

```diff
@@ -521,21 +521,19 @@
 		<true/>
 	</dict>
 	<key>NSCameraUsageDescription</key>
-	<string>Allows you to upload photographs and videos</string>
+	<string>Lets websites ask permission to use your camera for video calls, recording and uploading videos, or taking and uploading photos.</string>
 	<key>NSHumanReadableCopyright</key>
 	<string>Copyright © 2024 DuckDuckGo. All rights reserved.</string>
 	<key>NSLocationUsageDescription</key>
-	<string>Allows you to share your geolocation</string>
-	<key>NSLocationWhenInUseUsageDescription</key>
-	<string>Allows you to share your location</string>
```

Comment:
This one is not in use at all on macOS (we got a note about this from App Review team too).